tests/unit/OptionsListUtilsTest.js

```diff
@@ -250,7 +251,7 @@ describe('OptionsListUtils', () => {
         results = OptionsListUtils.getNewChatOptions(REPORTS_WITH_CONCIERGE, PERSONAL_DETAILS_WITH_CONCIERGE, '', true);
 
         // All the personalDetails should be returned minus the currently logged in user and Concierge
-        expect(results.personalDetails.length).toBe(_.size(PERSONAL_DETAILS_WITH_CONCIERGE) - 2);
+        expect(results.personalDetails.length).toBe(_.size(PERSONAL_DETAILS_WITH_CONCIERGE) - 2 - 5);
```

Comment:
5 is used quite a lot, can we add a test that returns less than 5 `recentReports`?

Reply:
@kakajann can you create a var for 5, something like `let maxRecentReports = 5;`

Reply:
> 5 is used quite a lot, can we add a test that returns less than 5 recentReports?

In [this line](https://github.com/Expensify/Expensify.cash/pull/2818/files#diff-44ebcab9bae929692fc02b00f963c7bf517c6fb6093d3a5aef5dbbc0eed30468R237), we are testing `recentReports` that returns less than 5.

> @kakajann can you create a var for 5, something like let maxRecentReports = 5;

Of course, I did add a file-level constant variable for maxRecentReports in my last push